src/components/Sentence/Sentence.js

```diff
@@ -33,20 +33,20 @@ class Sentence extends Component {
         let blankString = "_____"
```

Comment:
try using `const` for vars that do not change. JS has a different concept of consts than other languages. it promote s the use of it for general variable names. It will safe guard your variables from being modified accidentally.